Add `tldraw` Drawing block to the BP Hub

teenoh opened the pull request:
This PR adds the drawing canvas block to block hub

CiaranMn reviewed (dismissed):
Hey @teenoh – looking forward to getting this published :) 

I noticed a couple of things when previewing it in the Hub:

1. It still has the default image and icon - can we give it its own?

![Screenshot 2022-05-26 at 08 42 24](https://user-images.githubusercontent.com/37743469/170442174-a80c0c7f-be5e-466e-a86f-0b1d375eb63f.png)

2. If you select the pencil, change to readonly mode and then click on the screen, it crashes. Can you see if there's a quick fix for this please?

teenoh commented:
Thanks @CiaranMn!

1. I have added the preview image and icon. It looks like this now

![Screenshot 2022-05-31 at 09 53 01](https://user-images.githubusercontent.com/16928688/171134414-21f5d0f4-d3d7-4629-a8a8-6dd2b8b7078e.png)

2. I couldn't find an immediate fix for the bug especially since it's caused by an internal issue with tldraw. I have removed the `readOnly` functionality for now and have created an issue in tldraw's repo to track the bug. Plan to add it back once the bug has been fixed

nonparibus commented:
@teenoh Can you x-link the `tldraw` issue here? Thanks.

teenoh commented:
I just updated the description and preview image

![Screenshot 2022-06-06 at 13 27 49](https://user-images.githubusercontent.com/16928688/172181278-16fc0258-e284-4b8c-bd21-0444786dde69.png)

kachkaev commented on hub/@tldraw/drawing-canvas.json:
```diff
@@ -0,0 +1,6 @@
+{
+  "repository": "https://github.com/hashintel/hash.git",
+  "commit": "6a8932a439077e270fbb2945800981b9305f5b3d",
```
[This commit](https://github.com/hashintel/hash/commit/6a8932a439077e270fbb2945800981b9305f5b3d) is not on a main branch, let’s update it to, say:

```suggestion
  "commit": "70db9afee5cc5ce3c0e39af14f2699f0cf031963",
```

([the latest one ATM](https://github.com/hashintel/hash/commits/main))

teenoh replied:
Updated in [2178e23](https://github.com/blockprotocol/blockprotocol/pull/316/commits/2178e2318b1f18005f7a157822fbaa7d736a9ea0)
